### ISSUE TITLE
[Idea] better gathering

### ISSUE THREAD

**Issue by amirparsadd** [peter-hunt/skyblock#14]
When i do `mine potato 1 100` it floods the console so i have an idea:
Instead of repeating a message 100 times, it calculates the time for gathering 100 of any item, then while its gathering it just shows a progress bar that is like this:

Gathering <item> : [■■■□□□] +<allXpGatheredInThatTime> <skill> xp | time left : <timeLeft>

And it doesnt spam the progress bar, it just changes it in one line

**Comment by peter-hunt** [comment]
Good idea! I agree with you, and yes, it would be much easier to scroll back for info if it's like that. Unfortunately, I have yet to find a library that shows a progress bar like that. But I will look for a clear buffer on the line print later or print the configurable progress bar display fractions to make it!

**Comment by wendellmeset** [comment]
@peter-hunt 
may i suggest this?
https://github.com/tqdm/tqdm

It even has a built in function that calculates time of a for loop and displays the progress bar accordingly. It also is very customizable!

**Comment by peter-hunt** [comment]
That looks like a perfect library for the better gathering system! Thank you for that! I'll add this some time soon!
